src/CoreWlan/Enums.cs

```diff
@@ -116,4 +116,20 @@ public enum CWKeychainDomain : nuint_compat_int {
 		User = 1,
 		System = 2,
 	}
+
+	[Native]
+	public enum CWEventType : nuint_compat_int {
+		None = 0,
+		PowerDidChange = 1,
+		SSIDDidChange = 2,
+		BSSIDDidChange = 3,
+		CountryCodeDidChange = 4,
+		LinkDidChange = 5,
+		LinkQualityDidChange = 6,
+		ModeDidChange = 7,
+		ScanCacheUpdated = 8,
+		VirtualInterfaceStateChanged = 9,
+		RangingReportEvent = 10,
+		Unknown = int.MaxValue
```

Comment:
@halkar So it appears you hit a fun case we haven't hit before.

Please change it to be long.MaxValue and i'll have to do some additional work after this PR to fix up the generator to do the right thing.

I know the review's been a bit brutal, but we have to get everything right the first time since we take API compatibility very seriously.